src/sage/combinat/colored_permutations.py

```diff
@@ -1269,13 +1268,22 @@ def cycle_type(self):
         Return a pair of partitions of ``len(self)`` corresponding to the
         signed cycle type of ``self``.
 
-        A *negative cycle* is a cycle `C = (c_0, \ldots, c_{2k-1})` such that
-        `c_0 = c_k`. Any other cycle is positive. For any cycle `C`, we
-        ignore the cycle `-C` (provided this is a different cycle).
+        A *cycle* is a tuple `C = (c_0, \ldots, c_k)` with `\pi(c_i) = c_{i+1}`
```

Comment:
I'm afraid we have a typo here: it should probably be `c_{k-1}`.

Reply:
Good catch; fixed.